node/src/chain_spec.rs

```diff
@@ -369,7 +369,7 @@ fn genesis_config(
 			// Assign network admin rights.
 			key: Some(root_key),
 		},
-		teerex: TeerexConfig { allow_sgx_debug_mode: true },
+		teerex: TeerexConfig { allow_sgx_debug_mode: true, allow_skipping_attestation: true },
```

Comment:
We deemed this ok because it is noted in the enclave struct if it was skipped, didn't we? Otherwise, we might want to set a conservative default for production networks.

Reply:
my reasoning was:
On a production network, we'll set that flag specifically anyway
On a test network, we don't want to introduce friction and pitfalls

It will always be transparent, if RA was skipped

Reply:
Yeah, seems legitimate to me. 👍